cmds/ls/ls_test.go

```diff
@@ -58,19 +58,22 @@ func TestLs(t *testing.T) {
 	// Create an empty directory.
 	testDir := filepath.Join(tmpDir, "testDir")
 	os.Mkdir(testDir, 0700)
-	os.Chdir(testDir)
 
 	// Create some files.
-	os.Create("f1")
-	os.Create("f2")
-	os.Create("f3\nline 2")
-	os.Create(".f4")
-	os.Mkdir("d1", 0740)
-	os.Create("d1/f4")
+	os.Create(filepath.Join(testDir, "f1"))
+	os.Create(filepath.Join(testDir, "f2"))
+	os.Create(filepath.Join(testDir, "f3\nline 2"))
+	os.Create(filepath.Join(testDir, ".f4"))
+	os.Mkdir(filepath.Join(testDir, "d1"), 0740)
+	os.Create(filepath.Join(testDir, "d1/f4"))
 
 	// Table-driven testing
 	for _, tt := range tests {
-		out, err := exec.Command(execPath, tt.flags...).Output()
+		c := exec.Command(execPath, tt.flags...)
+		t.Logf("Dir is %v", testDir)
+		c.Dir = testDir
+		out, err := c.Output()
+		t.Logf("out :%v err: %v", out, err)
```

Comment:
Can we remove the debug printing? It's just noise for people running the test.

Reply:
but only if you do -v, right? it does not appear otherwise. Is that really a problem?